curiosrc/window/compute_do.go

```diff
@@ -25,12 +25,16 @@ import (
 	"github.com/filecoin-project/lotus/build"
 	"github.com/filecoin-project/lotus/chain/actors/builtin/miner"
 	"github.com/filecoin-project/lotus/chain/types"
+	"github.com/filecoin-project/lotus/lib/ffiselect"
 	"github.com/filecoin-project/lotus/storage/sealer"
 	"github.com/filecoin-project/lotus/storage/sealer/storiface"
 )
 
 const disablePreChecks = false // todo config
 
+// TODO Surface into config?
+const challengeReadTimeout = 0
```

Comment:
This is in the config, also we really should wire this up.

https://github.com/filecoin-project/lotus/blob/master/documentation/en/default-curio-config.toml#L246

Without this if some sectors are e.g. on a blocked nfs mount we'll fail all sectors in the partition instead of just the sectors on the blocking mount.